axum-extra/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+# Unreleased
+
+- **added:** Add `json!` for easy construction of JSON responses ([#2962]).
+
```

Comment:
```suggestion
- **added:** Add `json!` for easy construction of JSON responses ([#2962]).

[#2962]: https://github.com/tokio-rs/axum/pull/2962

```

Reply:
Thanks – somehow I thought it would link automatically 🤣

Reply:
Maybe it actually does within GitHub, but changelogs are also read locally from time to time 😄